client/me/concierge/book/info-step.js

```diff
@@ -66,11 +70,19 @@ class InfoStep extends Component {
 	}
 
 	render() {
-		const { signupForm: { firstname, lastname, message, timezone }, translate } = this.props;
+		const {
+			currentUserLocale,
+			signupForm: { firstname, lastname, message, timezone },
+			translate,
+		} = this.props;
+		const notice = ! isDefaultLocale( currentUserLocale )
+			? translate( 'All Concierge Sessions are in English (Spanish is not available).' )
```

Comment:
"Spanish is not available"? What if the local language is different than Spanish? :)

Reply:
Haha good point, totally missed it.